mdn/developer-portal
# 1681 Add blog link to header, move About link to footer + improve a11y

This changeset:

* Adds a link to Blog to the header, configured to go straight to Mozilla Hacks (for now - we'll change it to the final URL when that's decided)
* Moves the About link to the Footer
* Refactors the footer in line with advice from https://github.com/mdn/developer-portal/issues/1708 and also reduces the div soup

Has been cross-browser tested and all three main breakpoints checked - there are no regressions compared to production, but the layout is _slightly_ different now we're using the Protocol classes better. 

(Related issues #1681 and #1708)

## Screenshots

<img width="963" alt="Screenshot 2020-07-13 at 22 35 55" src="https://user-images.githubusercontent.com/101457/87362566-e5be2000-c566-11ea-9fe0-cf6629279ba9.png">
<img width="527" alt="Screenshot 2020-07-13 at 22 49 36" src="https://user-images.githubusercontent.com/101457/87362569-e8b91080-c566-11ea-90fe-1eadadf87a4d.png">

<img width="1646" alt="Screenshot 2020-07-14 at 00 33 19" src="https://user-images.githubusercontent.com/101457/87363708-a5ac6c80-c569-11ea-9d52-73b27b34a357.png">





## How to test

- Code is up on Staging
- Check the Blog link
- Check the About link in the footer
- @schalkneethling please review the footer for a11y improvements if you have time
